app/Models/BillDetail.php

```diff
@@ -43,4 +44,29 @@ public function product()
     {
         return $this->belongsTo('App\Models\Product');
     }
+
+    /**
+     * Get total daily amount
+     *
+     * @return int
+     */
+    public static function dailyTotalAmount()
+    {
+        return BillDetail::where('bills_details.created_at', '>=', DB::raw("concat(CURDATE(), ' 00:00:00')"))
```

Comment:
[rank C] wow, we are using initial time for many place, so I think we need arrange it again.